doc/sg/c_cpp_coding_style.md

```diff
@@ -254,11 +269,42 @@ Furthermore, designated initializers do not play well with type deduction and ov
 As such, they are forbidden in the following contexts:
 - Do not call overloaded functions with a designated initializer: `overloaded({ .foo = 0 })`.
   Instead, disambiguate with syntax like `T var = { .foo = 0 }; overloaded(var);`.
-- Do not use designated initializers in any place where they would be used for type defuction.
+- Do not use designated initializers in any place where they would be used for type deduction.
   This includes `auto`, such as `auto var = { .foo = 0 };`, and a templated argument in a template function.
 
 It is recommended to only use designated initializers with types which use C-style declarations.
 
+### Naming
+**Structs, Classes and methods**
+The names of structs, classes and methods must be in `CamelCase`.
+
+**Variable and class members naming**
+The names of variables (including function parameters) and data members must be in `lower_snake_case`. Data members of classes (but not structs) additionally have trailing underscores, unless the variable represents a constant which will follow the [rule](#function-enum-struct-and-typedef-naming).
+
+For example:
+```cpp
+// in animal.cpp
+public Class AnimalInfo{
+  ...
+private:
+  uint32_t number_of_paws_;
+  std::string name_;
+public:
+  static constexpr uint32_t kMaxNumOfPaws=100;
+
+  void SetAnimalName(std::string new_name);
+};
+```
+
+### Features
+The use of `C` features must be avoided in C++ code, such as:
```

Comment:
```suggestion
### Features

The use of `C` features must be avoided in C++ code, such as:
```

Reply:
Probably best to write the bullet points below the other way around, saying "Use [C thing] instead of [C++ thing]".